src/api/component.js

```diff
@@ -257,18 +236,13 @@ Component.prototype = Object.create(HTMLElement.prototype, {
     }
   },
 
-  // v0 and v1
+  // v1
```

Comment:
Ahh yep, should probably convert that to `// spec` maybe? It's nice to differentiate in the code what is Skate and what is spec'd. What should we do when v2 comes? Will possibly want to differentiate between those, in that case this is future-proof :D